packages/smb-server/src/server/broker.class.ts

```diff
@@ -0,0 +1,50 @@
+import { randomUUID } from 'crypto';
+import { createServer, Server } from 'net';
+import { Injectable } from 'peque-di';
+import { BrokerSocket, Command, EventService, SocketService } from 'peque-smb-commons/src';
```

Comment:
These should come from the workspace package dist
```suggestion
import { BrokerSocket, Command, EventService, SocketService } from 'peque-smb-commons';
```

Reply:
I am going to fix all the above.

Reply:
I will manage this in the next commits.